src/autorest-core/lib/configuration.ts

```diff
@@ -326,8 +326,8 @@ export class ConfigurationView {
                 const path = s.Position.path as string[];
                 if (path) {
                   this.Message({
-                    Channel: Channel.Warning,
-                    Text: `Could not find the exact path ${JSON.stringify(path)}`
+                    Channel: Channel.Debug,
```

Comment:
this is supposed to be visible! i.e. if you see noise because of this, it's because of bugs in rules so the noise is exactly what I intended with this. The noise is supposed to be removed by fixing the bugs instead of muting the noise ;)